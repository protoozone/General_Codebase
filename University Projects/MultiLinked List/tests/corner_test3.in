NEW 0
NEW 0
NEW 0
NEW 0
REMOVE 0
REMOVE 0
INSERT 0 0 lol
REMOVE 1
VIEW 1
VIEW-NESTED 1
DELETE 1 0
REMOVE 1
INSERT 2 0 lol
REMOVE 2 0
VIEW 2