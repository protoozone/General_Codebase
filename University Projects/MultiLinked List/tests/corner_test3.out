List 0: 
List 1: 
List 2: 
List 3: 
List 0 has been removed.

Number of lists: 3
List 1
List 2
List 3
INVALID COMMAND: REMOVE
List 0: lol
List 1 has been removed.

Number of lists: 2
List 0
List 2
List 3
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW-NESTED
INVALID COMMAND: DELETE
INVALID COMMAND: REMOVE
List 2: lol
INVALID COMMAND: REMOVE
INVALID COMMAND: VIEW